### ISSUE TITLE
Consider using cli-table3 for easier output reading

### ISSUE THREAD

**Issue by hailwood** [75lb/renamer#36]
Right now it can get pretty hard to follow the output from a rename action, especially when things get multiline, it would be awesome if we could get a table layout going on.

https://www.npmjs.com/package/cli-table3

![Cli table preview](http://i.imgur.com/sYq4T.png)

**Comment by 75lb** [comment]
Hi.. the only minor, cosmetic issue with console tables is that they don't accept streaming input. Renamer has a streaming output, writing logs as files are renamed. If i use a table, i'd need to wait for renamer to finish its work (which could take a while on slow machines with a lot of renames) before writing a console table.

[Renamer v1](https://github.com/75lb/renamer/tree/next) will land soon which has two new output views: `--view long` and `--view diff` (screenshot below). Do these new views help? 

![screen shot 2018-07-16 at 11 14 30](https://user-images.githubusercontent.com/1247980/42753643-7508aa12-88e9-11e8-91b9-30a57f118826.png)

**Comment by hailwood** [comment]
Yep, I actually recently grabbed `@next` after logging this issue, the new outputs are great
Would it be possible to have `Renamed: ✔` hidden unless `-v` is set? Given that, as far as I am aware, unless `-v` is set, every item output will be renamed?

**Comment by 75lb** [comment]
yeah, that would make sense.. i removed the `Renamed: ✔` output unless the `-v` flag is set, thanks. Let me know if you find anything else. You could submit a PR for `--view table` if you like, shouldn't be too hard to implement.
